src/main/java/dk/dbc/common/records/CatalogExtractionCode.java

```diff
@@ -163,9 +164,28 @@ static boolean hasPublishingDate(String value, List<String> listOfCatalogCodes)
         return result;
     }
 
-    static boolean hasPublishingDateIgnoreCatalogCodes(String value) {
+    /**
+     *
+     * @param code    the subfield name
+     * @param value   the subfield content
+     * @return bool   return a boolean result depending on :
+     * the name & returns false
+     * value OVE as katalog code returns false
+     * length different from 9 returns false
+     * value after catalog code that doesn't match certain date format returns false
+     * otherwise true
+     */
+    static boolean verifySubfieldAndContent(String code, String value) {
         logger.entry(value);
 
+        if ("&".equals(code)) {
+            return false;
+        }
+        final String catalogCode = value.substring(0, 3);
+        if ("OVE".equals(catalogCode)) {
```

Comment:
måske if(value.startsWith("OVE")) ville være bedre.

Reply:
yup - to sekunder